modules/ossm-threescale-webassembly-module-configuration.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assembly:
+//
+// service_mesh/v2x/ossm-threescale-webassembly-module.adoc
+
+[id="ossm-threescale-webassembly-module-configuration_{context}"]
+= The 3scale WebAssembly module configuration
+
+The `ServiceMeshExtension` custom resource spec provides the configuration that the `Proxy-WASM` module reads from.
```

Comment:
I notice in this module, Proxy-WASM is marked up with backticks and called a "module". Just checking whether that difference is intentional versus the earlier usage of italics and calling it a "specification".

Reply:
This is consistent with the upstream documentation. Is this correct @rahulanand16nov ?

Reply:
Both of them are the same thing but written in different styles. To make it more straightforward, we use `Proxy-WASM` spec to create these modules, so to say it in short, those modules are called `Proxy-WASM` modules.

Reply:
Thanks @rahulanand16nov